config/application.rb

```diff
@@ -27,6 +27,6 @@ class Application < Rails::Application
       'responsive/width-0-479.css',
     ]
     config.assets.debug = true
-    config.cache_store = :dalli_store
+    config.cache_store = :mem_cache_store
```

Comment:
This is the same compatibility-wise. Rails more or less just absorbed the adapter.